src/main/java/org/stellar/sdk/responses/AccountResponse.java

```diff
@@ -25,6 +25,10 @@
   private Long sequenceNumber;
   @SerializedName("subentry_count")
   private Integer subentryCount;
+  @SerializedName("sequence_ledger")
+  private Long sequenceUpdatedAtLedger;
+  @SerializedName("sequence_time")
+  private Long sequenceUpdatedAtTime;
```

Comment:
This should be a timestamp, no?

Reply:
I thought this was emitted[ from Horizon API json here](https://github.com/stellar/go/pull/4294/files#diff-491f1f1cafca2952be807956ff510ad82f7283582bad566e15b3e3ad00f451beR47) as integral Epoch right?